service/src/test/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextFlightTest.java

```diff
@@ -152,20 +151,16 @@ void createsProjectAndContext_emptySpendProfile_fails() throws Exception {
 
     assertEquals(FlightStatus.ERROR, flightState.getFlightStatus());
     assertTrue(workspaceService.getAuthorizedGcpCloudContext(workspaceId, userRequest).isEmpty());
-    String projectId =
-        flightState.getResultMap().get().get(WorkspaceFlightMapKeys.GCP_PROJECT_ID, String.class);
-    // The Project should exist, but requested to be deleted.
-    Project project = crl.getCloudResourceManagerCow().projects().get(projectId).execute();
-    assertEquals(projectId, project.getProjectId());
-    assertEquals("DELETE_REQUESTED", project.getState());
+    assertFalse(
+        flightState.getResultMap().get().containsKey(WorkspaceFlightMapKeys.GCP_PROJECT_ID));
   }
 
   @Test
   @DisabledIfEnvironmentVariable(named = "TEST_ENV", matches = BUFFER_SERVICE_DISABLED_ENVS_REG_EX)
-  void createsProjectAndContext_unauthorizedSpendProfile_fails() throws Exception {
+  void createsProjectAndContext_unauthorizedSpendProfile_flightFailsAndGcpProjectNotCreated()
+      throws Exception {
     UUID workspaceId = createWorkspace(spendUtils.defaultSpendId());
-    AuthenticatedUserRequest userRequest =
-        userAccessUtils.secondUserAuthRequest().email("liam.dragonmaw@test.firecloud.org");
+    AuthenticatedUserRequest userRequest = userAccessUtils.secondUserAuthRequest();
```

Comment:
nit: to make it clearer that this is not the primary test user, could you rename this variable something like `secondaryUserRequest` or `unauthorizedUserRequest`?